ckanext/canada/templates/public/package/read.html

```diff
@@ -238,6 +238,11 @@ <h4>{{ _('Contact Information') }}</h4>
   {% endif %}
 {% endblock contact_information %}
 
+
+{% block package_rate %}
+{% include "package/rate.html" %}
+{% endblock %}
```

Comment:
let's put this one line in directly instead of adding a new `package/rate.html` template to `{% include %}` (we could do the same for the one below too, it used to be a large block of logic but is just an ajax call not)